lib/swoosh/x_test_assertions.ex

```diff
@@ -64,6 +66,24 @@ defmodule Swoosh.X.TestAssertions do
     end
   end
 
+  @doc ~S"""
+  Asserts no emails were sent.
+  """
+  @spec assert_no_email_sent() :: boolean | no_return
+  def assert_no_email_sent() do
+    refute_email_sent()
+  end
+
+  @doc ~S"""
+  Asserts `email` was not sent.
+
+  Performs exact matching of the email struct.
+  """
+  @spec assert_email_not_sent(Email.t()) :: boolean | no_return
+  def assert_email_not_sent(%Email{} = email) do
+    refute_email_sent(email)
+  end
+
```

Comment:
note to myself: might still want to remove these to simply favor the refute functions.